web/packages/design/src/theme/themes/bblpTheme.ts

```diff
@@ -0,0 +1,205 @@
+limitations under the License.
+*/
+
+import { lighten } from '../utils/colorManipulator';
+
+import { sharedColors, sharedStyles } from './sharedStyles';
+import { DataVisualisationColors, Theme, ThemeColors } from './types';
+
+const dataVisualisationColors: DataVisualisationColors = {
+  primary: {
+    purple: '#9F85FF',
+    wednesdays: '#F74DFF',
+    picton: '#009EFF',
+    sunflower: '#FFAB00',
+    caribbean: '#00BFA6',
+    abbey: '#FF6257',
+    cyan: '#00D3F0',
+  },
+  secondary: {
+    purple: '#7D59FF',
+    wednesdays: '#D50DE0',
+    picton: '#007CC9',
+    sunflower: '#AC7400',
+    caribbean: '#008775',
+    abbey: '#DB3F34',
+    cyan: '#009CB1',
+  },
+  tertiary: {
+    purple: '#B9A6FF',
+    wednesdays: '#FA96FF',
+    picton: '#7BCDFF',
+    sunflower: '#FFD98C',
+    caribbean: '#2EFFD5',
+    abbey: '#FF948D',
+    cyan: '#74EEFF',
+  },
+};
+
+const colors: ThemeColors = {
+  ...sharedColors,
+
+  levels: {
+    deep: '#000000',
+
+    sunken: '#191919',
+
+    surface: '#232323',
+
+    elevated: '#282828',
+
+    popout: '#373737',
+  },
+
+  spotBackground: [
+    'rgba(255,255,255,0.07)',
+    'rgba(255,255,255,0.13)',
+    'rgba(255,255,255,0.18)',
+  ],
+
+  brand: '#FFA028',
+
+  text: {
+    main: '#FFFFFF',
+    slightlyMuted: '#BEBEBE',
+    muted: '#8C8C8C',
+    disabled: '#646464',
+    primaryInverse: '#000000',
+  },
+
+  buttons: {
+    text: '#FFFFFF',
+    textDisabled: 'rgba(255, 255, 255, 0.3)',
+    bgDisabled: 'rgba(255, 255, 255, 0.12)',
+
+    primary: {
+      text: '#000000',
+      default: '#FFA028',
+      hover: '#FFB04C',
+      active: '#DB8922',
+    },
+
+    secondary: {
+      default: 'rgba(255,255,255,0.07)',
+      hover: 'rgba(255,255,255,0.13)',
+      active: 'rgba(255,255,255,0.18)',
+    },
+
+    border: {
+      default: 'rgba(255,255,255,0)',
+      hover: 'rgba(255, 255, 255, 0.07)',
+      active: 'rgba(255, 255, 255, 0.13)',
+      border: 'rgba(255, 255, 255, 0.36)',
+    },
+
+    warning: {
+      text: '#FFFFFF',
+      default: '#E51E3C',
+      hover: '#FD2D4A',
+      active: '#C31834',
+    },
+
+    trashButton: {
+      default: 'rgba(255, 255, 255, 0.07)',
+      hover: 'rgba(255, 255, 255, 0.13)',
+    },
+
+    link: {
+      default: '#66ABFF',
+      hover: '#99C7FF',
+      active: '#2B8EFF',
+    },
+  },
+
+  tooltip: {
+    background: '#212B2F',
+  },
+
+  progressBarColor: '#00BFA5',
+
+  error: {
+    main: '#E51E3C',
+    hover: '#FD2D4A',
+    active: '#C31834',
+  },
+
+  warning: {
+    main: '#FA5A28',
+    hover: '#FB754C',
+    active: '#D64D22',
+  },
+
+  notice: {
+    background: '#282828', // elevated
+  },
+
+  action: {
+    active: '#FFFFFF',
+    hover: 'rgba(255, 255, 255, 0.1)',
+    hoverOpacity: 0.1,
+    selected: 'rgba(255, 255, 255, 0.2)',
+    disabled: 'rgba(255, 255, 255, 0.3)',
+    disabledBackground: 'rgba(255, 255, 255, 0.12)',
+  },
+
+  terminal: {
+    foreground: '#F1F2F4',
```

Comment:
I just noticed that it should just be `FFF`, it looks like a mistake from when I implemented light / dark themes. 
I asked Yassine to fix this here, and in `darkTheme` while we are here.

The difference is minimal, `#F1F2F4` looks like ~95% white, so I think no one noticed it.